asset/js/ets_woo_qa_script.js

```diff
@@ -36,7 +36,7 @@ jQuery(document).ready(function(){
     	e.preventDefault();
     	let clickedButton = jQuery(this);
     	let productId = clickedButton.parent().find("[name='sh-prd-id']").val();
-    	let formPrdId = jQuery('#custId').val();
+    	let formPrdId = jQuery('.custId').val();
```

Comment:
What is the diff b/w `productId` and `formPrdId`?

Reply:
**productId**  is the  shortcode hidden input value, **formPrdId** gets from the form hidden input value